kod/object/passive/moon.kod

```diff
@@ -41,13 +47,34 @@ messages:
       Send(self,@NewGameHour);
       propagate;
    }
+      
+   GetMoonPhase(dayOfYear = $)
+   {
+      local scaledLunarMonth, phaseLength, currentLunarDay;
+      scaledLunarMonth = 19;
+      currentLunarDay = dayOfYear mod scaledLunarMonth;
+      if currentLunarDay >= 1 AND currentLunarDay <= 5
```

Comment:
To reduce the amount of magic numbers hard-coded in here, how about doing calculations based on "scaledLunarMonth" instead?

Also:
- 19 doesn't divide 240, so there will be weirdness around New Years.  Should scaledLunarMonth be 20 instead?
- From day 0 to day 1 we go eighth -> full?  Seems like it should go eighth -> quarter -> half -> full -> half -> quarter -> eighth (given we don't have the art assets to do the crescent correctly on both sides).
- You mentioned that the non-full assets are twice the size, and it looks weird in the screenshots.